cpu/cc2538/radio/cc2538_rf_radio_ops.c

```diff
@@ -323,6 +322,7 @@ void cc2538_irq_handler(void)
         /* CRC check */
         uint8_t pkt_len = rfcore_peek_rx_fifo(0);
         if (rfcore_peek_rx_fifo(pkt_len) & CC2538_CRC_BIT_MASK) {
+            RFCORE_XREG_RFIRQM0 &= RXPKTDONE;
```

Comment:
This only disables the ISR not overriding the frame buffer content, IMO something like https://github.com/RIOT-OS/RIOT/blob/338a11a16f3476d9cffbc6381bf0e0b21ad5c838/cpu/cc2538/radio/cc2538_rf_radio_ops.c#L345-L346 is more accurate, in  #15145 I was re-enabling after reception reading the frame buffer, but I guess with the new logic this is not needed.

Reply:
You are right. I will try that

Reply:
Or just take a look at #15145 :D